library/init/meta/congr_tactic.lean

```diff
@@ -33,7 +33,8 @@ meta def congr_core : tactic unit :=
 do tgt ← target,
    apply_eq_congr_core tgt
    <|> apply_heq_congr_core
-   <|> fail "congr tactic failed"
+   <|> fail "congr tactic failed",
+   all_goals (try (applyc `subsingleton.elim))
```

Comment:
Is this line still necessary?

Reply:
Ah, I thought I got rid of that line. Time to remove it and recompile mathlib again :grimacing: